.github/workflows/release_dockerhub_image.yml

```
@@ -145,90 +172,124 @@ jobs:
         run: >
           breeze release-management release-prod-images --dockerhub-repo "${REPOSITORY}"
           --airflow-version "${AIRFLOW_VERSION}" ${SKIP_LATEST}
-          --limit-python ${PYTHON_VERSION} --limit-platform linux/amd64
+          --python ${PYTHON_MAJOR_MINOR_VERSION}
+          --metadata-folder dist
       - name: >
-          Verify regular AMD64 image: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}
-        env:
-          PYTHON_VERSION: ${{ matrix.python-version }}
-          AIRFLOW_VERSION: ${{ github.event.inputs.airflowVersion }}
-          REPOSITORY: ${{ github.repository }}
+          Verify regular image: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}, ${{ matrix.platform }}
         run: >
-          breeze prod-image verify
-          --pull
-          --image-name
-          ${REPOSITORY}:${AIRFLOW_VERSION}-python${PYTHON_VERSION}
+          breeze prod-image verify --pull --manifest-file dist/${MANIFEST_FILE_NAME}
       - name: >
-          Release slim images AMD64 first: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}
-        env:
-          COMMIT_SHA: ${{ github.sha }}
-          REPOSITORY: ${{ github.repository }}
-          PYTHON_VERSION: ${{ matrix.python-version }}
-          AIRFLOW_VERSION: ${{ github.event.inputs.airflowVersion }}
-          SKIP_LATEST: ${{ needs.build-info.outputs.skipLatest == 'true' && '--skip-latest' || '' }}
+          Release slim images: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}, ${{ matrix.platform }}
         run: >
           breeze release-management release-prod-images --dockerhub-repo "${REPOSITORY}"
           --airflow-version "${AIRFLOW_VERSION}" ${SKIP_LATEST}
-          --limit-python ${PYTHON_VERSION} --slim-images --limit-platform linux/amd64
+          --python ${PYTHON_MAJOR_MINOR_VERSION} --slim-images
+          --metadata-folder dist
       - name: >
-          Verify slim AMD64 image: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}
-        env:
-          PYTHON_VERSION: ${{ matrix.python-version }}
-          AIRFLOW_VERSION: ${{ github.event.inputs.airflowVersion }}
-          REPOSITORY: ${{ github.repository }}
+          Verify slim image: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}, ${{ matrix.platform }}
         run: >
-          breeze prod-image verify
-          --pull
-          --slim-image
-          --image-name
-          ${REPOSITORY}:slim-${AIRFLOW_VERSION}-python${PYTHON_VERSION}
-      - name: >
-          Release regular images for both AMD / ARM: ${{ github.event.inputs.airflowVersion }}, ${{ matrix.python-version }}
+          breeze prod-image verify --pull --slim-image --manifest-file dist/${MANIFEST_SLIM_FILE_NAME}
+      - name: "Upload metadata artifacts"
```

```suggestion
      - name: "Upload metadata artifacts: ${{ env.ARTIFACT_NAME }}"
```

Just to see step name with artifact that uploaded :)

---

Yep....